bin/import-or-require.js

```diff
@@ -8,7 +8,7 @@ module.exports = function importOrRequire(file) {
     const ext = extnamePath(file);
 
     if (ext === '.mjs' || (ext === '.js' && getPackageType.sync(file) === 'module')) {
-        return import(file);
+        return import("file://" + file);
```

Comment:
this will break if someone uses `--require=packageName`. You need to provide the `flle://` on relative ESM paths yourself, I think.

Reply:
I am still not sure about the `--import=packageName` (only find the require and ignore option). Could you please give an example that would break with this fix?